README.md

```diff
@@ -78,29 +78,25 @@ Once the node template is running locally, you can connect to it with frontends
 
 ## How to upgrade to new Polkadot release
 
-**Note: Now that this repo has upgraded to using dependencies from crates.io, this section
-needs to be updated to reflect the new process, once the first release of the crates from
-the new `polkadot-sdk` mono-repo happens**
-
-- [ ] Check Substrate's [`node-template`](https://github.com/paritytech/substrate/commits/master/bin/node-template)
+- [ ] Check Substrate's [`node-template`](https://github.com/paritytech/polkadot-sdk/tree/master//substrate/bin/node-template/),
+      which was renamed after the Polkadot release `1.8.0` to [`solochain-template`](https://github.com/paritytech/polkadot-sdk/tree/master/templates/solochain),
```

Comment:
@pgherveou should we also add https://github.com/paritytech/polkadot-sdk/tree/master/templates/parachain since we have support for parachain mode?

Reply:
Yes we probably should